reference/promise-types/packages.markdown

```diff
@@ -443,3 +443,19 @@ Manage packages using [pkgsrc](https://www.pkgsrc.org).
 **History:**
 
 * Added in CFEngine 3.9.0
+
+### slackpkg
+
+Manage packages using Slackware's [slackpkg](https://slackpkg.org).
+
+**Example**
+
+```cf3
+  packages:
+    slackware::
+      "nmap"
+        policy => "absent",
+        package_module => slackpkg;
+```
+
+**History:**
```

Comment:
Looks like we'd want something like "Added in CFEngine 3.12.0" for history.

Reply:
Done !